### PR TITLE
add AMR

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** info added around new AMR option for IDPs
- **Is this PR related to a Monolith release?** yes 2023.05.2

Netlify:

OktatoOkta: https://644ad26af781570d3c3ec8bd--reverent-murdock-829d24.netlify.app/docs/guides/add-an-external-idp/oktatookta/main/#create-an-identity-provider-in-okta
OpenIDConnect: https://644ad26af781570d3c3ec8bd--reverent-murdock-829d24.netlify.app/docs/guides/add-an-external-idp/openidconnect/main/#create-an-app-at-the-identity-provider
SAML2: https://644ad26af781570d3c3ec8bd--reverent-murdock-829d24.netlify.app/docs/guides/add-an-external-idp/saml2/main/#create-an-app-at-the-identity-provider

### Resolves:

* [OKTA-593253](https://oktainc.atlassian.net/browse/OKTA-593253)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/azure/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/965f8584966f56a0c78363c924dccc2f3ccf1057/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fazure%2Fappidpinokta.md) | [77](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7d97d2ed-3d6b-4124-ab90-62bff5630ce6) | 59  | 100  | 59  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/965f8584966f56a0c78363c924dccc2f3ccf1057/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Foktatookta%2Fappidpinokta.md) | [77](https://okta.acrolinx.cloud/api/v1/checking/scorecards/37d9f8bc-7372-4483-bd6a-d1305e6f42d4) | 64  | 71  | 62  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/965f8584966f56a0c78363c924dccc2f3ccf1057/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fopenidconnect%2Fappidpinokta.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/315d9972-572c-4c1d-923e-08cce5e93dc4) | 73  | 67  | 80  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/965f8584966f56a0c78363c924dccc2f3ccf1057/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fsaml2%2Fappidpinokta.md) | [66](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a1043bf0-7d86-4f5d-be68-85c1e425224b) | 40  | 69  | 44  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.cf5ea83e-b8d6-462d-8f3c-6a2d19540a6f?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/azure/appidpinokta.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 * **Name**: Enter a name for the Identity Provider configuration.
 * **Client Id**: Paste the client ID that you obtained from Azure AD when you configured the Identity Provider in the previous section.
 * **Client Secret**: Paste the secret that you obtained in the previous section.
+* **Authentication method reference (AMR) claims**: Select **Trust AMR claims from this identity provider** to have Okta evaluate any AMR claims sent in the request from the IdP to meet sign-on policy requirements.
```
Comment (shantanusardal-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/azure/appidpinokta.md:4]: this won't be applicable to Azure IdP

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/08ba1923670c9d9593e08bc29de5ffbde4a3902f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Foktatookta%2Fappidpinokta.md) | [73](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1967a01f-f402-4f3b-85de-157dafa0d956) | 64  | 71  | 62  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/08ba1923670c9d9593e08bc29de5ffbde4a3902f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fopenidconnect%2Fappidpinokta.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/01ba95be-367b-4b4b-b002-e0622ad69f97) | 73  | 67  | 80  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/08ba1923670c9d9593e08bc29de5ffbde4a3902f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fsaml2%2Fappidpinokta.md) | [63](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f206e303-f28f-441c-a451-2782b8e536ba) | 40  | 69  | 44  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.5086a8e3-9c08-4c52-87a4-58feeecf95ab?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by shantanusardal-okta — APPROVED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/529691765be6df070ae17a741887bfe503be1a77/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Foktatookta%2Fappidpinokta.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/02bfff5e-c7b9-4d84-aa95-6058809657c5) | 78  | 71  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/529691765be6df070ae17a741887bfe503be1a77/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fopenidconnect%2Fappidpinokta.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4917a619-dcc6-46f9-baaa-c41c68e50a3c) | 80  | 67  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/529691765be6df070ae17a741887bfe503be1a77/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fsaml2%2Fappidpinokta.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/17774ede-9cd4-4c0b-af6e-ffcc3c974bb1) | 42  | 68  | 77  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.461dfc1c-ebb3-4f37-b4a0-ac3f5c55510a?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/c696c052ef82b4435341dc606b3ea7a8f4b4a6f5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Foktatookta%2Fappidpinokta.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3aed1637-6d08-4345-afd1-0e8dd2929f45) | 78  | 71  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/c696c052ef82b4435341dc606b3ea7a8f4b4a6f5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fopenidconnect%2Fappidpinokta.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/190479f6-1092-47ae-b61c-bc9608fdc84d) | 80  | 67  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md](https://github.com/okta/okta-developer-docs/blob/c696c052ef82b4435341dc606b3ea7a8f4b4a6f5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fsaml2%2Fappatidp.md) | [68](https://okta.acrolinx.cloud/api/v1/checking/scorecards/59475c4e-c55f-454f-bcfe-4eb1d21d5479) | 61  | 76  | 61  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/c696c052ef82b4435341dc606b3ea7a8f4b4a6f5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fsaml2%2Fappidpinokta.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5756ff2f-53be-4c38-a53f-e6d5ae7efd39) | 42  | 68  | 77  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.2c6712b4-4b18-407a-a81e-29d34ac793d4?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by susanharper-okta** [comment]
approved by Nick Gamb in Slack.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/36d7c99d451b3992f53e641506b62879e35722cc/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Foktatookta%2Fappidpinokta.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1ba0b758-e877-4d5e-bbb3-a35d5a5b7cf6) | 78  | 71  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/36d7c99d451b3992f53e641506b62879e35722cc/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fopenidconnect%2Fappidpinokta.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ac8f5412-fe5d-4c79-be70-4f1b8904984b) | 80  | 67  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md](https://github.com/okta/okta-developer-docs/blob/36d7c99d451b3992f53e641506b62879e35722cc/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fsaml2%2Fappatidp.md) | [68](https://okta.acrolinx.cloud/api/v1/checking/scorecards/83cd9e87-9f96-4285-bf79-4d499b7f3c62) | 61  | 76  | 61  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/36d7c99d451b3992f53e641506b62879e35722cc/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fsaml2%2Fappidpinokta.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c6302dca-f1c2-4063-a759-9487ede7e6df) | 42  | 68  | 77  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.7461347e-0e68-44e8-8146-6908f8a42dec?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/a131eca5036f8a2be151f6d93657281eedd17e9b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Foktatookta%2Fappidpinokta.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2818cb0c-7a55-4a7c-8d8d-984e628b92f0) | 78  | 71  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/a131eca5036f8a2be151f6d93657281eedd17e9b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fopenidconnect%2Fappidpinokta.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4a900873-85b4-46a4-8148-a4ee6aec5427) | 80  | 67  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md](https://github.com/okta/okta-developer-docs/blob/a131eca5036f8a2be151f6d93657281eedd17e9b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fsaml2%2Fappatidp.md) | [68](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4524b04c-a325-4edf-ab4a-3838092c02c8) | 61  | 76  | 61  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/a131eca5036f8a2be151f6d93657281eedd17e9b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fsaml2%2Fappidpinokta.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3e96f4b1-c9e2-466b-b07a-8401a77bb571) | 42  | 68  | 77  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.7b151921-7009-40aa-98c4-3449c22c1cc7?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by shantanusardal-okta — APPROVED** [review]

**Review by nakuljoshi-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
-Access the external SAML Identity Provider that you want to integrate with and gather the following information for use in configuring the SAML Identity Provider in Okta in the next step. Make sure that the SAML Identity Provider supports Service Provider-initiated SAML.
+Access the external SAML Identity Provider (IdP) you want to integrate with and gather the following information. You use this information to configure the SAML Identity Provider in Okta in the next step. Make sure ßthe SAML IdP supports Service Provider-initiated SAML.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md:1]:
```suggestion
Access the external SAML Identity Provider (IdP) you want to integrate with and gather the following information. You use this information to configure the SAML Identity Provider in Okta in the next step. Make sure the SAML IdP supports Service Provider-initiated SAML.
```

**Review by susanharper-okta — COMMENTED** [review]
typo

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/492a90e9b5dfc55d24c1e4d75793834063ff6b8b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Foktatookta%2Fappidpinokta.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5a1c1813-1822-4e83-a033-d366c26db582) | 78  | 78  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/492a90e9b5dfc55d24c1e4d75793834063ff6b8b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fopenidconnect%2Fappidpinokta.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/720b5861-5fec-4cfd-8780-a8e6d78e5794) | 80  | 73  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md](https://github.com/okta/okta-developer-docs/blob/492a90e9b5dfc55d24c1e4d75793834063ff6b8b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fsaml2%2Fappatidp.md) | [71](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6887f0e1-5d69-4557-b7da-a4085950544f) | 61  | 76  | 61  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appidpinokta.md](https://github.com/okta/okta-developer-docs/blob/492a90e9b5dfc55d24c1e4d75793834063ff6b8b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-an-external-idp%2Fmain%2Fsaml2%2Fappidpinokta.md) | [74](https://okta.acrolinx.cloud/api/v1/checking/scorecards/abdb30f2-984c-493c-b2f4-d0840a17f25d) | 42  | 77  | 77  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.fd1f4c9e-8844-473c-a540-515a60a02874?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Pull request merged**